test/util/assertRovingTabindex.js

```diff
@@ -30,7 +30,7 @@ module.exports = async function assertRovingTabindex (t, elementsSelector, key)
       assert.equal(
         await elements[el].getAttribute('tabindex'),
         tabindex,
-        'focus is on element ' + tabableEl + ' of elements "' + elementsSelector +
+        'focus is on element ' + tabableEl + ' of ' + elements.length + ' elements "' + elementsSelector +
```

Comment:
@jongund, @sh0ji, this is now different from latest from master ... I don't understand this change. Since I merged those other fixes in from Valerie, is this change helpful/needed?